### PR TITLE
Update with the 1.3.4 version

### PR DESCRIPTION
**Pull request by stilobique**
## Problem

- Update with the 1.3.4 version

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

File: manifest.toml [hunk 1]
```diff
@@ -44,15 +44,15 @@ ram.runtime = "50M"
 [resources]
     [resources.sources]
         [resources.sources.api]
-        url = "https://dev.funkwhale.audio/funkwhale/funkwhale/-/jobs/artifacts/1.3.3/download?job=build_api"
+        url = "https://dev.funkwhale.audio/funkwhale/funkwhale/-/jobs/artifacts/1.3.4/download?job=build_api"
         sha256 = "1c2906d968089a796752f9444dd777146d5f71c4af3d2f083740759c2ef656fa"
```
Comment (Thovi98) [inline manifest.toml:48]: Thank you!
I think your forgot to update the SHA sum here?

**Comment by Thovi98** [comment]
May I also ask you to update the `tests.toml`?

Thanks for the PR!

File: manifest.toml (hunk 1 above)
Comment (stilobique) [reply manifest.toml]: Haha, i'm look the old hash file, fixed !

**Comment by stilobique** [comment]
> May I also ask you to update the `tests.toml`?
> 
> Thanks for the PR!

Is simply the commit identifier ? It's add.

**Review by Thovi98 — APPROVED** [review]
Very nice, thanks!

**Comment by Thovi98** [comment]
!testme
!bookwormtestme

**Comment by Thovi98** [comment]
@stilobique May I ask you where you took the commit number from in tests.toml?
It has to be the latest commit refernence in master branch :)
I have `88dd43d`

**Comment by Thovi98** [comment]
Okay, for reference, the commit didn't work because we forgot to update the package version in manifest.toml :)

**Comment by stilobique** [comment]
> Okay, for reference, the commit didn't work because we forgot to update the package version in manifest.toml :)

Oh, I think I mixed up the branches when I took the commit ID.
Thanks for your explain @Thovi98 , the next time I make a PR without issue 🔥 !

**Pull request merged**